invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookMetadata/BookMetadata.js

```diff
@@ -0,0 +1,274 @@
+import { RequestNewLoanForm } from './components/RequestNewLoanForm';
+import '../../BookDetails.scss';
+
+export default class BookMetadata extends Component {
+  state = { visible: false };
+
+  toggleVisibility = () => this.setState({ visible: !this.state.visible });
+
+  _render_book_info() {
+    const bookData = this.props.bookDetails;
+    return (
+      <div className="book-info">
+        <Header as="h2">{bookData.title}</Header>
+        <List>
+          {bookData.authors.map((author, index) => (
+            <List.Item as="h4" key={`Key${index}`}>
+              Author: {author}
+            </List.Item>
+          ))}
+        </List>
+        <List>
+          {bookData.publishers.map((publisher, index) => (
+            <List.Item as="h5" key={`Key${index}`}>
+              Publisher: {publisher}
+            </List.Item>
+          ))}
+        </List>
+      </div>
+    );
+  }
+
+  _render_share_buttons_large() {
+    return (
+      <div>
+        <Responsive minWidth={768}>
+          <Button
+            href="https://www.facebook.com/sharer/sharer.php"
+            color="facebook"
+          >
+            <Icon name="facebook" /> Facebook
+          </Button>
+          <div className="ui hidden divider" />
+          <Button href="https://twitter.com/intent/tweet" color="twitter">
+            <Icon name="twitter" /> Twitter
+          </Button>
+          <div className="ui hidden divider" />
+          <Button color="linkedin">
+            <Icon name="linkedin" /> LinkedIn
+          </Button>
+        </Responsive>
+      </div>
+    );
+  }
+
+  _render_files() {
+    const files = this.props.bookDetails.files;
+    return (
+      <div>
+        <Header as="h3">Files</Header>
+        <List>
+          {files.map((file, index) => (
+            <List.Item href={file} key={`Key${index}`}>
+              {file}
+            </List.Item>
+          ))}
+        </List>
+      </div>
+    );
+  }
+
+  _render_links() {
+    const booklinks = this.props.bookDetails.booklinks;
+    return (
+      <div>
+        <Header as="h3">Links</Header>
+        <List>
+          {booklinks.map((link, index) => (
+            <List.Item href={link} key={`Key${index}`}>
+              {link}
+            </List.Item>
+          ))}
+        </List>
+      </div>
+    );
+  }
+
+  _render_attachments() {
+    return (
+      <Grid.Column width={3}>
+        <Grid.Row>
+          <Header as="h3">Share and Export</Header>
+          {this._render_share_buttons_large()}
+          <div className="ui hidden divider" />
+          <div className="ui hidden divider" />
+          <div className="ui divider" />
+        </Grid.Row>
+
+        <Grid.Row>
+          {this._render_files()}
+          <div className="ui divider" />
+        </Grid.Row>
+
+        <Grid.Row>{this._render_links()}</Grid.Row>
+      </Grid.Column>
+    );
+  }
+
+  _render_share_buttons() {
+    return (
+      <Responsive maxWidth={768}>
```

Comment:
`768` magic number, and I see further down is used again. Create the constants you need, so they are consistent all across the project. We can also change them all at once, if our screens get bigger.

Reply:
My opinion is that we should not set any width anywhere, but we should trust the library that already investigated on screen sizes.
In Semantic UI, you have predefined breakpoints if I am not wrong:
https://react.semantic-ui.com/addons/responsive/#usage-breakpoints

I suggest to use that ones instead, and remove any `switch width` config. WDYT?